src/QueryDescription/index.js

```diff
@@ -174,6 +175,10 @@ export function sanitizeLikeString(value: string): string {
   return value.replace(nonLikeSafeRegexp, '_')
 }
 
+export function textMatches(value: string): Comparison {
```

Comment:
one more place that takes Queries is `encodeMatcher` (used by query observation system). I think it would be appropriate to throw an error if you try to encode a matcher for a query that contains the unsupported textMatches operator. (Even better would be to support it, or to make .observe() de-opt to reloadingObserver, but that's out of scope for this PR)